zerver/webhooks/dialogflow/tests.py

```diff
@@ -6,35 +6,39 @@ class DialogflowHookTests(WebhookTestCase):
     FIXTURE_DIR_NAME = "dialogflow"
 
     def test_dialogflow_default(self) -> None:
+        email = self.example_user("aaron").email
```

Comment:
Isn't this just `self.example_email("aaron")`?

Reply:
That's aaron's real email (`.delivery_email`). `.email` is the dummy email because in tests we now have public email visibility disabled.

Reply:
Ahh, OK, sgtm.